Sources/StreamChat/APIClient/APIClient.swift

```diff
@@ -4,6 +4,16 @@
 
 import Foundation
 
+/// The retry policy to use by the API retry mechanism
+public struct RetryOptions {
+    /// How many attempts should be performed at most
+    let maxRetries: Int = 3
+    /// How much time there should be in between attempts, this function is usually dependant on the amount of attempts already performed
+    let backoff: (Int) -> Double = {
```

Comment:
This helps better understand what is the `Int` 
```suggestion
    let backoff: (maxRetries: Int) -> Double = {
```